front_end/core/host/RNPerfMetrics.ts

```diff
@@ -17,6 +17,7 @@ export function getInstance(): RNPerfMetrics {
 type UnsunscribeFn = () => void;
 class RNPerfMetrics {
   #listeners: Set<RNReliabilityEventListener> = new Set();
+  #launchId: string|null = null;
```

Comment:
nit / question: Do we have Prettier enabled on TS files?

Reply:
Yes. This is the formatted output, consistent with the upstream formatting, e.g.:

https://github.com/facebookexperimental/rn-chrome-devtools-frontend/blob/9370b35e1979ec6953ff59ed78c8751e4f463750/front_end/core/sdk/Cookie.ts#L19

The `npm` lint or `git cl format` commands are hit-or-miss tho. Tried running the linter in #12 which created divergent output vs upstream main.

The docs says the project format is `clang`:

https://docs.google.com/document/d/1WNF-KqRSzPLUUfZqQG5AFeU_Ll8TfWYcJasa_XGf7ro/view#heading=h.4lu5vvp3iz1w

https://chromium.googlesource.com/devtools/devtools-frontend/+/main/docs/README.md#useful-commands

So I'm running this formatter locally on save:

https://marketplace.visualstudio.com/items?itemName=xaver.clang-format

And my workspace-specific settings to avoid changing unmodified lines:
```json
// .vscode/settings.json
{
  …
  "editor.codeActionsOnSave": {
    "source.fixAll": true
  },
  "editor.formatOnSaveMode": "modificationsIfAvailable",
  "[typescript]": {
    "editor.defaultFormatter": "xaver.clang-format"
  }
  …
}
```

Reply:
Clang does TypeScript?? what

Reply:
Yeah and Google is very proud of it in the doc above

> The JS code style of the DevTools is basically the Blink Coding Style Guidelines, which are for C++, but we'll apply them to JS. (Yup)